subjects/java/checkpoints/singleton-blueprint/README.md

```diff
@@ -0,0 +1,41 @@
+## Singleton Blueprint
+
+### Instructions
+
+You are given an incomplete singleton class. Complete the class to demonstrate your understanding of how the Singleton design pattern works. The Singleton pattern ensures that a class has only one instance and provides a global point of access to that instance.
+
+### Expected Class
+
+```java
+public class Singleton {
+    public Singleton instance;
+
+    private Singleton() {
+        // Initialization code
+    }
+
+    public Singleton get???() {
+        // Implementation to return the single instance
+    }
+}
+```
+
+### Usage
+
+Here is a possible `ExerciseRunner.java` to test your class:
+
+```java
+public class ExerciseRunner {
```

Comment:
Not returning anything, i think you missed 
 ```
 singleton = Singleton.getInstance();
showMessage();
```

Reply:
yeah you are right!